File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        String answer = "";
        char n1 = my_string.charAt(num1);
        char n2 = my_string.charAt(num2);
        for (int i = 0; i < my_string.length(); i++) {
            if (i == num1) {
                answer = answer + n2;
            } 
            else if (i == num2) {
                answer = answer + n1;
            } 
            else {
                answer = answer + my_string.charAt(i);
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    
    for i in range(len(my_string)) :
        if i == num1 :
            answer += my_string[num2]
        elif i == num2 : 
            answer += my_string[num1]
        else :
            answer += my_string[i]
    return answer